src/app/shared/types/faction.type.ts

```diff
@@ -2,9 +2,10 @@ import { TableRow } from './general';
 
 export const FACTION_TABLE = 'factions'; // sqlite
 export const FACTION_ID = 'm_ID';
-export const FACTION_SEARCH_FIELDS = [FACTION_ID, 'm_name_lang_1'];
+export const FACTION_SEARCH_FIELDS = [FACTION_ID, 'faction_name_id', 'm_name_lang_1'];
```

Comment:
here you need to change the value of the faction_ID to select the faction_name_id instead of the m_ID when the user select a faction through the faction selector

```typescript
export const FACTION_ID = 'faction_name_id';
export const FACTION_SEARCH_FIELDS = ['m_ID', 'faction_name_id', 'm_name_lang_1'];
```

Reply:
The problem is that on the faction text field we need to have the m_ID, because its the same as the info we get from the sniffs.
Im going to add the data using the sqlitebrowser and see how it goes. If i have to change something, with real data will be easier i think.